runway/hooks/staticsite/auth_at_edge/templates/check_auth/__init__.py

```diff
@@ -0,0 +1,249 @@
+# pylint: skip-file
```

Comment:
it would be better to address the linting errors then to exclude the entire file